src/main/java/com/minecolonies/coremod/client/gui/WindowBuildTool.java

```diff
@@ -397,15 +399,6 @@ private void hutDecClicked(@NotNull final Button button)
      */
     private List<String> getStyles()
     {
-        try
```

Comment:
Okay that would be nice. Its very nice if you can scan and then build on the go without having to restart.